tests/snuba/sessions/test_sessions_v2.py

```diff
@@ -222,6 +224,51 @@ def test_virtual_groupby_query():
     assert query.query_groupby == []
 
 
+class SnubaSessionsV2Test(TestCase, SnubaTestCase):
+    # This test suite needs a project to exist in the database.
```

Comment:
I considered a better alternative to create this class over mocking plenty of things. Easier to read and maintain, imo.